modules/account-lib/test/unit/coin/cspr/transaction.ts

```diff
@@ -170,6 +284,36 @@ describe('Cspr Transaction', () => {
       });
     });
 
+    it('wallet initialization using extended key', async function() {
+      const walletInitTx = await getWalletInitTransactionUsignExtendedKey();
+      const encodedTx = walletInitTx.toBroadcastFormat();
+      const walletInitJsonTx = JSON.parse(encodedTx);
+
+      const argName = 0;
+      const argValue = 1;
+      const owner0 = 0;
+      const owner1 = 1;
+      const owner2 = 2;
```

Comment:
nit: You could use a regular for loop instead of creating 3 consts to use in `forEach`